yarn.lock

```diff
@@ -59,6 +87,16 @@
     "@algolia/requester-common" "4.8.3"
     "@algolia/transporter" "4.8.3"
 
+"@algolia/client-analytics@4.8.6":
```

Comment:
@hobadams how come we get these duplicates?

Reply:
@Borales - good spot, this has been resolved.

I had to run  `npx yarn-deduplicate`